models/users.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: [true, 'Name is required'],
+  },
+  email: {
+    type: String,
+    required: [true, 'Email is required'],
+    unique: true,
+  },
+  password: {
+    type: String,
+    required: [true, 'Set password for user'],
+  },
+  subscription: {
+    type: String,
+    enum: ['starter', 'pro', 'business'],
+    default: 'starter',
+  },
+  owner: {
+    type: Schema.Types.ObjectId,
+    ref: 'user',
+  },
```

Comment:
To przynależy dla kontaktów. Bo to miejsce w kontaktach będzie przetrzymywac informacje o uzytkowniku do którego nalezy ta baza danych :-)